QuartzCore/CAFilter.h

```diff
@@ -1,58 +1,156 @@
-/**
- * This header is generated by class-dump-z 0.2-1.
- * class-dump-z is Copyright (C) 2009 by KennyTM~, licensed under GPLv3.
- *
- * Source: /System/Library/Frameworks/QuartzCore.framework/QuartzCore
- */
-
-#import "QuartzCore-Structs.h"
+#import <Foundation/Foundation.h>
 #import <QuartzCore/CABase.h>
 
+API_AVAILABLE(ios(2.0))
 @interface CAFilter : NSObject <NSCopying, NSMutableCopying, NSCoding>
 
 @property (readonly) NSString *type;
 @property (copy) NSString *name;
 @property (getter=isEnabled) BOOL enabled;
-@property BOOL cachesInputImage;
-// ["multiplyColor","multiplyGradient","gaussianBlur","pageCurl","fog","lighting","clear","copy",
-//  "sourceOver","sourceIn","sourceOut","sourceAtop","destOver","destIn","destOut","destAtop","xor","plusL","multiply"]
+@property BOOL cachesInputImage API_AVAILABLE(ios(3.0));
+
 + (NSArray <NSString *> *)filterTypes;
 + (CAFilter *)filterWithType:(NSString *)type;
-+ (CAFilter *)filterWithName:(NSString *)name;
++ (CAFilter *)filterWithName:(NSString *)name; // alias for filterWithName:
```

Comment:
alias for filterWithType: ?

Reply:
yes, that makes more sense. thanks (just double-checked in Hopper as well)